CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ ENHANCEMENTS:
 BUG FIXES:
 * Fix numbering issue within `bug_report.md` template ([#4028](https://github.com/microsoft/AzureTRE/pull/4028))
 * Disable public network access to the API App Service ([#3986](https://github.com/microsoft/AzureTRE/issues/3986))
+* Fix Guacamole shared drive always enabled ([#4046](https://github.com/microsoft/AzureTRE/pull/4046))
```

Comment:
wrong issue url?

Reply:
```suggestion
* Fix Guacamole shared drive always enabled ([#3885](https://github.com/microsoft/AzureTRE/issues/3885))
```

This better?

I'm thinking now GitHub will auto generate release notes (that we can amend), do we still need a changelog? Separate disucssion!